internal/mithril-persistence/src/sqlite/connection_extensions.rs

```diff
@@ -115,4 +123,56 @@ mod tests {
 
         assert_eq!(value, 45);
     }
+
+    #[test]
+    fn test_apply_execute_the_query() {
```

Comment:
Would it be relevant to ensure correct behavior with `update` and `delete` queries as well?

Reply:
I'm not sure that does much. The apply function only executes the query whatever it is.
We can add this to the test but it will make it more complex. 
Do you think this is useful?

Reply:
I see your point and I'm fine keeping it as is. 👍